src/Microsoft.AspNetCore.Mvc.Abstractions/Formatters/InputFormatterExceptionModelStatePolicy.cs

```diff
@@ -1,11 +1,51 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+
 namespace Microsoft.AspNetCore.Mvc.Formatters
 {
+    /// <summary>
+    /// Defines the set of policies that determine how the model binding system interprets exceptions
+    /// thrown by an <see cref="IInputFormatter"/>. Applications should set 
+    /// <c>MvcOptions.InputFormatterExceptionModelStatePolicy</c> to configure this setting.
```

Comment:
Remember to update these references to the name as well (because they might not be found by a regular symbol rename).

Reply:
👍

Reply:
This should actually be a `<see ...>` not a `<c>`. Will fix.

Reply:
NVM my BRAIN is crazy. Fixed it